scripts/mixins/weapon_break.lua

```diff
@@ -23,7 +23,7 @@ g_mixins.weapon_break = function(weaponBreakMob)
             local animationSub = mob:getAnimationSub()
 
             -- break weapon
-            if animationSub == 0 then
+            if animationSub ~= 1 then
```

Comment:
Not very familiar with this mixin, but shouldn't 3 be included here as well if both weapons are broken?

Reply:
This is for mobs with only one weapon